src/net.cpp

```diff
@@ -2994,10 +2997,7 @@ CNode::CNode(NodeId idIn, ServiceFlags nLocalServicesIn, SOCKET hSocketIn, const
     m_serializer = std::make_unique<V1TransportSerializer>(V1TransportSerializer());
 }
 
-CNode::~CNode()
-{
-    CloseSocket(hSocket);
-}
+CNode::~CNode() {}
```

Comment:
pls move to header and use `= default` (or document specific reason why it must be in cpp)

Reply:
I Just removed the destructor, it is not necessary now.